File: task-03/Go/code2..go
package main

import (
    "io/ioutil"
    "os"
)

func main() {
    // Read from input.txt and write to output.txt
    content, err := ioutil.ReadFile("input.txt")
    if err != nil {
        panic(err)
    }
    err = ioutil.WriteFile("output.txt", content, 0644)
    if err != nil {
        panic(err)
    }
}

File: task-03/Javascript/code2..js
const fs = require('fs');

// Read from input.txt and write to output.txt
fs.readFile('input.txt', 'utf8', (err, data) => {
    if (err) throw err;
    fs.writeFile('output.txt', data, err => {
        if (err) throw err;
    });
});
